ocaml/idl/datamodel.ml

```diff
@@ -1184,7 +1184,8 @@ let _ =
     ~doc:"The uploaded update package is invalid" ();
   error Api_errors.update_is_applied [ ]
     ~doc:"The specified update has been applied and cannot be destroyed." ();
-
+  error Api_errors.cannot_find_update []
+    ~doc:"The requested update VDI could not be found. This can occur when you run xe update-pool-clean. Please upload the update again" ();
```

Comment:
Thanks @lindig, how about `The requested update could not be found`? I think `VDI` is confusing in the message, also there's `Please upload the update again` in the message as guidance.
Does it make sense?